consensus/bor/heimdall.go

```diff
@@ -14,5 +15,9 @@ type IHeimdallClient interface {
 	Span(ctx context.Context, spanID uint64) (*span.HeimdallSpan, error)
 	FetchCheckpoint(ctx context.Context, number int64) (*checkpoint.Checkpoint, error)
 	FetchCheckpointCount(ctx context.Context) (int64, error)
+	FetchMilestone(ctx context.Context) (*milestone.Milestone, error)
+	FetchMilestoneCount(ctx context.Context) (int64, error)
+	FetchNoAckMilestone(ctx context.Context, milestoneID string) (bool, error)
```

Comment:
Could you add a comment on what NoAsk stands for?

Reply:
Do we need (bool, error)? Could we stick with just error in return?

Reply:
Sure,I will add the comment. It is basically the rejected milestone. And I used the term NoAck because we use similar terminology in checkpoint modules for failed checkpoint.